src/ert/scheduler/job.py

```diff
@@ -178,6 +168,19 @@ async def _max_runtime_task(self) -> None:
         )
         self.returncode.cancel()
 
+    async def _handle_finished_forward_model(self) -> None:
+        callback_status, status_msg = forward_model_ok(self.real.run_arg)
+        if self._callback_status_msg != "":
```

Comment:
Isn't this reversed? Ie. `if self._callback_status_msg == ""` ?

Reply:
If so, it is not related to this PR (I think we have discussed this before, and I also think the current behaviour is required by tests)

Reply:
Yes, we had this discussed this before but it still poses a question for me.

Reply:
Yes, you will fail a test in at least tests/integration_tests/status/test_tracking_integration.py if you change to ==